server/athenian/api/align/goals/dbaccess.py

```diff
@@ -265,11 +265,22 @@ async def get_goal_template_from_db(template_id: int, sdb: DatabaseLike) -> Row:
     return template
 
 
-async def get_goal_templates_from_db(account: int, sdb: DatabaseLike) -> list[Row]:
+async def get_goal_templates_from_db(
+    account: int,
+    sdb: DatabaseLike,
+    exclude_with_metric_params: bool,
```

Comment:
```suggestion
    exclude_with_metric_params: bool,
    sdb: DatabaseLike,
```
we tend to specify the DB args last, unless there are optional kwargs